.github/workflows/link-check.yaml

```
@@ -18,7 +18,7 @@ jobs:
         id: lc
         uses: peter-evans/link-checker@v1
         with:
-          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|.*.md" .
+          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|https://asktug.com/|https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC|http://www.alhem.net/project/mysql/|https://www.figma.com/file/dHQ9B7X4mF1q78NRIzNTs5/tidb-sketch-book-2020|http://www.webgraphviz.com/|http://octave.sourceforge.net/database/index.html|http://gnade.sourceforge.net/|.*.md" .
```

```suggestion
          args: -c 32 -d /github/workspace -r -x "http://172.*|https://github.com/.*/pull/.*|https://github.com/.*/issues/.*|https://github.com/[^.\n]*$|http://127.0.0.1.*|http://localhost.*|http://.*:.*|https://static.googleusercontent.com/media/research.google.com/.*|http://www.brendangregg.com/perf.html.*|https://blog.csdn.net.*|http://mysql-cocoa.sourceforge.net.*|https://tools.ietf.org.*|http://code.google.com.*|https://spark.apache.org.*|https://asktug.com/*|https://shimo.im/sheets/*|http://www.alhem.net/project/mysql/|http://www.webgraphviz.com/|http://octave.sourceforge.net/database/index.html|http://gnade.sourceforge.net/|.*.md" .
```
I don't have the permission to access https://www.figma.com/file/dHQ9B7X4mF1q78NRIzNTs5/tidb-sketch-book-2020.

---

+ I searched the repo and did not find this link now. Let's ignore this one.
+ For the shimo.im link, if other shimo.im link is added to the repo and later becomes 404, the link checker will be unable to detect the 404 link.

---

👌